packages/create-sitecore-jss/src/templates/vue/src/lib/layout-service-factory.js

```diff
@@ -0,0 +1,34 @@
+import {
+  LayoutService,
+<% if (fetchWith.toUpperCase() === 'GRAPHQL') {
+  -%>
```

Comment:
closing tags should be on same line as opening tags

Reply:
As soon as I save the file they move to the next line.
Is it a problem if they are on different lines?

Reply:
Maybe your editor or some prettier setting may be seeing them as JSX, and adding the last bracket as a new line formatting? That's what it looks like to me if it's on save.

> Is it a problem if they are on different lines?

Not sure if the ejs render will work, also we want it consistent with the other frameworks regardless.